tools/api-go-structs-requirements.txt

```diff
@@ -1,3 +1,3 @@
-Jinja2==2.10
+Jinja2>=2.10.1
```

Comment:
Mopidy-Dirble ~= 1.1    we could imho use this `~= ` for not taking a 3 major version

Reply:
https://pip.readthedocs.io/en/stable/reference/pip_install/#requirements-file-format